src/powershell/Public/Initialize-FinOpsHubDeployment.ps1

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+<#
+    .SYNOPSIS
+    Initialize a FinOps hub deployment in order to enable resource group owners to deployment hubs via the portal.
+
+    .PARAMETER WhatIf
+    Optional. Shows what would happen if the command runs without actually running it.
+
+    .EXAMPLE
+    Initialize-FinOpsHubDeployment `
+    [-WhatIf]
+
+    Shows what would happen if the command runs without actually running it.
+
+    .Description
+    The Initialize-FinOpsHubDeployment command performs any initialization tasks required for a resource group contributor to be able to deploy a FinOps hub instance in Azure, like registering resource providers. To view the full list of tasks performed, run the command with the -WhatIf option.
+#>
+
+
+function Initialize-FinOpsHubDeployment {
+    [CmdletBinding(SupportsShouldProcess)]
+    param()
+
+    if ($PSCmdlet.ShouldProcess('Initialization Tasks for FinOpsHub')) {
```

Comment:
Instead of us checking `ShouldProcess()` here, should we just pass `-WhatIf` thru to the commands we're calling? I'm fine either way. What do y'all think?

If we keep it here, we should change this to be more explicit. Also, Aren't there 2 parameters?
```suggestion
    if ($PSCmdlet.ShouldProcess('Required resource providers', 'Register')) {
```